build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
```

Comment:
In general I am pretty sure quiet is meant to mean "no output in the success case", but in the error case we still want nice error messages.